octane.md

```diff
@@ -137,6 +137,23 @@ services:
 
 You may consult [the official FrankenPHP documentation](https://frankenphp.dev/docs/docker/) for more information on running FrankenPHP with Docker.
 
+<a name="frankenphp-configuration"></a>
+## Configuring FrankenPHP
+
+When using FrankenPHP with Laravel, note that the default system `php.ini` does not apply to the FrankenPHP worker environment. To configure PHP settings, create a `php.ini` file in your project's root directory:
```

Comment:
This is not exact. The system default is taken into account, but FrankenPHP uses the default system path (hardcoded in the PHP source code) which is `/lib/php.ini`. Most distributions change this default path to `/etc`, hence the confusion.

If you use the static binary (default with Octane), you can use `/lib/php.ini` or the root directory of the project.  If you use the Docker image, you should use `$PHP_INI_DIR`. If you compile PHP by yourself, you can specify the path you want by passing ` --with-config-file-path` option to `./configure`.

This is documented here: https://frankenphp.dev/docs/config/

Reply:
Thanks @dunglas, I've rewritten to hopefully be a little more precise on this matter. Are you able to take a look and check if you think it covers it?